src/core/comparable.ml

```diff
@@ -5,3 +5,9 @@ module type S = sig
 
   val compare:t Comparator.t
 end
+
+module type C = sig
```

Comment:
I would avoid adding this definition. I intentionally removed it from Imm.re, because the definition needs to be considered within the scope of modular implicits.